src/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,95 @@
+export chow_ring, select
+
+function chow_ring(M::Matroid, extended=false)
+	Flats = flats(M)
+	sizeFlats = size(Flats)[1]
+	n = size(M.groundset)[1]
```

Comment:
I would suggest to use ncols and nrows instead, so:
```suggestion
	n = nrows(M.groundset)
```